data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,57 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+mission "Coalition Folklore: Saryd"
+	minor
+	invisible
+	to offer
+		random < 3
+	source
+		attributes "saryd" "folklore"
+	on offer
+		conversation
+			`Walking by the spaceport, you're approached by a Saryd in unusually traditional clothing, even by Saryd standards, with silk robes not too different from human kimonos covering most of his body, and thin ropes tightly wrapping around it near his hoofs.`
```

Comment:
This feels like it should be broken into two sentences.
I also find the end confusing. What does "it" refer to?

Reply:
It refers to the clothing, though now that you pointed that out, "wrapping around them" or "wrapping around the robes" would probably be better.
Where are you suggesting the break into two sentences?